### PR TITLE
Add Get-AzVirtualRouterPeerAdvertisedRoute and Get-AzVirtualRouterPeerLearnedRoute cmdlets

### PR DESCRIPTION
**Pull request by litchiyangMSFT**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.VirtualRouterTests/TestVirtualRouterPeerCRUD.json [hunk 1]
```diff
@@ -4478,7 +4478,7 @@
     }
   ],
   "Names": {
-    "Test-VirtualRouterPeerCRUD": [
+    "Test-HubBgpConnectionCRUD": [
```
Comment (isra-fel) [inline src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.VirtualRouterTests/TestVirtualRouterPeerCRUD.json:4481]: It seems you modified the test case name in recordings but not source code, so the case is failing

> [xUnit.net 00:08:19.35]     Commands.Network.Test.ScenarioTests.VirtualRouterTests.TestVirtualRouterPeerCRUD [FAIL]

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.VirtualRouterTests/TestVirtualRouterPeerCRUD.json (hunk 1 above)
Comment (litchiyangMSFT) [reply src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.VirtualRouterTests/TestVirtualRouterPeerCRUD.json]: resolved

**Review by isra-fel — APPROVED** [review]

**Comment by isra-fel** [comment]
As we talked offline, network-july is not going into this milestone, so could you please change the base branch to master? Thanks

**Comment by isra-fel** [comment]
Close on behalf of #13196

**Pull request closed without merging**